File: Pruebas_Analizador_Semantico/codigosSoloJava/FibonacciRecur.py
def fibonacci(n):
    if n <= 1:
        return n
    else:
        return fibonacci(n - 2) + fibonacci(n - 1)

if __name__ == "__main__":
    print("Print Fibonacci Series Using Recursion in Python")

    MAX = 10
    for i in range(MAX):
        print(fibonacci(i), end=" ")

File: Pruebas_Analizador_Semantico/codigosSoloJava/FibonacciRecur.java
public class FibonacciRecur{

	
	public static int fibonacci(int MAX) {

		
		if (MAX <= 1) {
			return MAX;
		}

		
		else {
			
			return fibonacci(MAX - 2) + fibonacci(MAX - 1);
		}

	}

	public static void main(String[] args) {

		System.out.println("Print Fibonacci Series Using Recursion in Java");

		int MAX = 10;
		for (int i = 0; i < MAX; i++) {
		    System.out.print(fibonacci(i) + " ");
		}
	}
}